src/contexts/reducers/authReducer.js

```diff
@@ -0,0 +1,18 @@
+const authReducer = (state, action) => {
+  switch (action.type) {
+    case "LOGIN":
+      return { ...state, user: action.payload.user, isAuthenticated: true };
+
+    case "SIGN_UP":
+      return { ...state, user: action.payload.user, isAuthenticated: true };
+
+    case "LOGOUT":
+      localStorage.removeItem("token");
+      return { ...state, isAuthenticated: false, user: {} };
+
+    default:
+      throw new Error(action.type, "Wrong action type");
```

Comment:
you can return state as it is in the default scenario

Reply:
Fixed it, thanks